tools/cmake_test.py

```diff
@@ -265,8 +265,21 @@ def _run(self):
                      args=self.test_args)
         logger.info(cmd)
 
+        # setup llvm symbolizer. first look for location in ci, then in redpanda
+        # vbuild directory. if none, then asan will look in PATH
+        env = os.environ.copy()
+        llvm_symbolizer = shutil.which("llvm-symbolizer", path="/vectorized/llvm/bin")
+        if llvm_symbolizer is None:
+            path = os.path.join(os.path.dirname(os.path.abspath(__file__)), "..", "vbuild/llvm/install/bin")
+            path = os.path.abspath(path) # remove ".."
+            llvm_symbolizer = shutil.which("llvm-symbolizer", path=path)
+        if llvm_symbolizer is not None:
+            env["ASAN_SYMBOLIZER_PATH"] = llvm_symbolizer
+        sys.stderr.write(f"Using llvm-symbolizer: {llvm_symbolizer}\n")
```

Comment:
Is this debugging output that got left in? Elsewhere we seem to use `logger.foo` so maybe this should be converted to that if it is going to stay in?

Reply:
Yeh. Not sure why there are two logging mechanisms in there. I've updated it to use `logger.info`:

```
7: INFO:2022-10-14 15:18:58,569 cmake_test.py:278] Using llvm-symbolizer: /home/nwatkins/src/redpanda/vbuild/llvm/install/bin/llvm-symbolizer                                                                                                                                                                                                              
```

Change:

https://github.com/redpanda-data/redpanda/compare/cf74f5b266093f3fcf9c102688016e22b60342c3..2c3fdf23be1a118401b60d87e8fc2f36c667bdf9